src/dragonball/src/api/v1/vmm_action.rs

```diff
@@ -634,3 +634,626 @@ fn handle_cpu_topology(
 
     Ok(cpu_topology)
 }
+#[cfg(test)]
```

Comment:
Nit: Add a blank line between code and test mod.

Reply:
fixed